test/tasks/MultisigTask.t.sol

```diff
@@ -135,8 +135,10 @@ contract MultisigTaskUnitTest is Test {
             abi.encode(bytes32(uint256(100)))
         );
 
+        address[] memory safes = new address[](1);
+        safes[0] = multisig;
         vm.expectRevert("MultisigTask: hash mismatch");
-        task.simulate("", actions);
+        task.simulate("", actions, safes);
```

Comment:
Above comment means this needs to be `simulateRun` and we should be able to avoid this interface change. There's another test that also calls `task.simulate` we should change

Reply:
fixed: https://github.com/ethereum-optimism/superchain-ops/pull/1079/commits/8dc7da72c853a76686192e4036743a77b95dca12